environments.json

```diff
@@ -6712,5 +6712,13 @@
     "equals": "chrome104",
     "release": "2022-09-02",
     "obsolete": false
+  },
+  "reactnative0_70_3": {
+    "full": "React Native 0.70.3 (Using bundled Hermes and metro-react-native babel preset)",
```

Comment:
I think that it should specify the used Hermes version.

Reply:
Hermes version is same as react native release now `Hermes release version: for RN 0.70.3`.

![image](https://user-images.githubusercontent.com/430289/200047538-4c383c31-5edb-466e-971f-701f32a2410a.png)

https://github.com/facebook/react-native/releases/tag/v0.70.3

Reply:
And [what version](https://github.com/facebook/hermes/releases) is this mean?

Reply:
Those are standalone Hermes release, as you can see it is not that frequent release but bundled Hermes is released faster along with each RN release. This means we have newer version of Hermes with RN 🎉  and more ES support (hopefully!).

Version will show like `Hermes release version: 0.12.0` (see the difference from previous comment) for non-bundled Hermes version

![image](https://user-images.githubusercontent.com/430289/200051181-78a394a1-4cbb-4074-bd80-8d624475d52f.png)

Reply:
Ok.

Reply:
I updated summary for documentation with output of this conversation, great question and thanks for raising it.